packages/mds-geography-author/api.ts

```diff
@@ -58,8 +59,8 @@ function api(app: express.Express): express.Express {
         ) {
           params.get_published = true
         }
-        const metadata = await db.readBulkGeographyMetadata(params)
-        return res.status(200).send(metadata)
+        const geography_metadata = await db.readBulkGeographyMetadata(params)
+        return res.status(200).send({ version: res.locals.version, geography_metadata })
```

Comment:
To be clear ... this is a _breaking_ change. The structure of the payload is different. It used to be just the metadata, now it is an object containing the metadata and the version. One option would be to define the versions as `['0.0.0', '0.1.0']`; making `0.0.0` the default and only return the structured response when the version _isn't_ `0.0.0`. Something to think about if we are trying to avoid breaking changes.

Reply:
I don't think we have a choice here, given what the 0.4.1 specification says.  Should we do a quick huddle to discuss?